### DISCUSSION TITLE
Where is QueryClient in v4?

### DISCUSSION THREAD

**Discussion by MasonGomm** [TanStack/query#5024] [General] [answered]
When I try to do this `import { QueryClient } from '@tanstack/react-query'` it says it doesn't exist. Everything in the docs show this is how they are importing, but it is not working for me. Extremely frustrating. I'm on version 4.24.10.

**Comment by TkDodo** [comment, marked as answer]
Yes, that's how it works. Have a look at the examples, they do it that way, too.
